packages/angular/ssr/public_api.ts

```diff
@@ -10,4 +10,9 @@ export {
   CommonEngine,
   type CommonEngineRenderOptions,
   type CommonEngineOptions,
-} from './src/common-engine';
+} from './src/common-engine/common-engine';
+
+// TODO(alanagius): enable at a later stage
+// export { AngularAppEngine } from './src/app-engine';
+// export { AngularServerApp } from './src/app';
+// export { REQUEST, REQUEST_CONTEXT, RESPONSE_INIT } from './src/tokens';
```

Comment:
At this point, the API isn't yet usable because it requires the Angular CLI to inject and create the manifest. This will be addressed in a follow-up pull request.